qt/place_page_dialog_user.cpp

```diff
@@ -0,0 +1,291 @@
+#include "qt/place_page_dialog_user.hpp"
+
+#include "qt/qt_common/text_dialog.hpp"
+
+#include "map/place_page_info.hpp"
+#include "indexer/validate_and_format_contacts.hpp"
+#include "platform/settings.hpp"
+
+#include <QtWidgets/QDialogButtonBox>
+#include <QtWidgets/QGridLayout>
+#include <QtWidgets/QHBoxLayout>
+#include <QtWidgets/QLabel>
+#include <QtWidgets/QPushButton>
+
+#include <qboxlayout.h>
+#include <qdialog.h>
+#include <qgridlayout.h>
+#include <qnamespace.h>
+#include <sstream>
+#include <string>
+
+const int place_page_description_max_length = 500;
+const int short_description_minimum_width = 390;
```

Comment:
No global vars please. Put them into an anonymous namespace.
```suggestion
int constexpr kMaxLengthOfPlacePageDescription = 500;
int constexpr kMinWidthOfShortDescription = 390;
```

Reply:
done